lib/public/components/Filters/common/filtersPanel.js

```diff
@@ -44,7 +44,7 @@ export const filtersPanel = (globalModel, filteringModel, columns) => h(
             }
             return accumulator;
         }, []),
-        h('.p2', h('button.btn.btn-danger.mt2', {
+        h('.p2', h('#reset-filters.button.btn.btn-danger.mt2', {
```

Comment:
As `button` is the html element, it needs to go first. Only after you follow with IDs and Classes